MobiFlight/InputConfig/ButtonInputConfig.cs

```diff
@@ -69,20 +77,36 @@ public void WriteXml(System.Xml.XmlWriter writer)
             writer.WriteStartElement("onRelease");
             if (onRelease != null) onRelease.WriteXml(writer);
             writer.WriteEndElement();
+
+            writer.WriteStartElement("onLongRelease");
+            if (onLongRelease != null) onLongRelease.WriteXml(writer);
+            writer.WriteEndElement();
         }
 
         internal void execute(CacheCollection cacheCollection, 
                               InputEventArgs args, 
                               List<ConfigRefValue> configRefs)
         {
-            if (args.Value == 0 && onPress != null)
+            var inputEvent = (MobiFlightButton.InputEvent)args.Value;
+            if (inputEvent == MobiFlightButton.InputEvent.PRESS && onPress != null)
             {
                 onPress.execute(cacheCollection, args, configRefs);
             }
-            else if (args.Value == 1 && onRelease != null)
+            else if (inputEvent == MobiFlightButton.InputEvent.RELEASE && onRelease != null)
             {
                 onRelease.execute(cacheCollection, args, configRefs);
             }
+            else if (inputEvent == MobiFlightButton.InputEvent.LONG_RELEASE)                

```

Comment:
i would prefer to move the second if case to the other condition above... so we would have onRelease.execute only once.

When test become longer, i started to create a method for the test with a speaking name. e.g. ```IsLongReleaseButOnlyReleaseDefined()```

and this is implemented as 

```return inputEvent == MobiFlightButton.InputEvent.LONG_RELEASE && onLongRelease==null && onRelease!=null```

Reply:
Done